code/game/gamemodes/wizard/artifacts.dm

```diff
@@ -128,9 +128,25 @@
 /datum/spellbook_artifact/summon_magic/purchased(mob/living/carbon/human/H)
 	..()
 
-	H.rightandwrong(1)
+	H.rightandwrong("magic")
 	to_chat(H, "<span class='userdanger'>You have shared the gift of magic with everyone.</span>")
 
+//SUMMON SWORDS
+/datum/spellbook_artifact/summon_swords
+	name = "Summon Swords"
+	desc = "Launch a crusade or just spark a blood bath. Either way there will be limbs flying and blood spraying."
+	abbreviation = "SS"
+
+/datum/spellbook_artifact/summon_magic/can_buy()
+	//Can't summon swords during ragin' mages
+	return !ticker.mode.rage
+
+/datum/spellbook_artifact/summon_swords/purchased(mob/living/carbon/human/H)
+	..()
+
+	H.rightandwrong("swords")
+	to_chat(H, "<span class='userdanger'>DEUS VULT!</span>")
```

Comment:
>not "I AM THE BONE OF MY SWORD"

Reply:
>we need anime references<
"no"

Reply:
Are there any other well-known abilities based entirely around summoning a bunch of swords?
It's a pretty good fit as far as references go, I don't personally think it should be overlooked just because it's an anime.

Reply:
I was trying to stick to the crusade theme with the armor and all. The phrase there isnt an incantation at all, just a phrase to show you bought it.

Reply:
Oh, I didn't realize at first that it was shown only to the user. I think it's fine as it is, then.

Reply:
Might as well either change the spell name to reflect the crusading thing or just drop the armor because they make the spell literal meme tier. These armors aren't just for show and are very robust.

Reply:
I don't think you actually looked at any armor when you made that post
Sec armor(you spawn with as an officer)
armor = list(melee = 50, bullet = 15, laser = 50, energy = 10, bomb = 25, bio = 0, rad = 0)
Knight armor
armor = list(melee = 40, bullet = 5, laser = 5, energy = 5, bomb = 0, bio = 0, rad = 0)
On top of that, I made the description of the spell about crusading already.

Reply:
RULES OF NATURE

Reply:
If I remove the armor I will likely go with that sonix.